docs/utils.rst

```diff
@@ -2,9 +2,20 @@
 Utilities
 =========
 
-The `pastastore.util` submodule contains functions for deleting libraries or
-entire databases. Functions are available for deleting content from both arctic
-and pystore implementations.
+The `pastastore.util` submodule contains useful functions, i.e. for deleting
+databases, connector objects, and PastaStore objects or emptying a library of
+all its contents:
 
-* :meth:`pastastore.util.delete_arctic`
-* :meth:`pastastore.util.delete_pystore`
+
+* :meth:`pastastore.util.delete_pastastore`
+* :meth:`pastastore.util.delete_dict_connector`
```

Comment:
what happens when you delete a connector? Why would you want to do that?

Reply:
It actually deletes the contents that the connector links to. Either a single library (oseries, stresses or models) or everything all at once. Perhaps a better name would clear that up, but don't have a better one yet.